src/EventListener/LocaleListener.php

```diff
@@ -29,9 +29,9 @@ public function onKernelRequest(RequestEvent $event): void
     {
         $request = $event->getRequest();
 
-        if (!$request->hasPreviousSession()) {
-            return;
-        }
+        // if (!$request->hasPreviousSession()) {
```

Comment:
Why do we need to comment this out?

Reply:
I commented this out while testing. Removed